scripts/tests/calibnet_other_check.sh

```diff
@@ -46,12 +46,11 @@ $FOREST_CLI_PATH sync wait # allow the node to re-sync
 
 echo "Test IPLD traversal by fetching the state of epoch 1"
 # The IPLD graph for the state-root of epoch 1 contains 1197 CIDs
-EXPECTED_WALK="IPLD graph traversed! CIDs: 1195, fetched: 12, failures: 0."
+EXPECTED_WALK="IPLD graph traversed! CIDs: 1195, fetched: 11, failures: 0."
```

Comment:
I removed this test for now, see https://github.com/ChainSafe/forest/issues/3209